File: Consultorio.Server/Models/Armazon.cs
namespace Consultorio.Server.Models
{
    public class Armazon
    {
        public int armazonid { get; set; }

        public string marca { get; set; } = string.Empty;

        public string modelo { get; set; } = string.Empty;

        public string color { get; set; } = string.Empty;

        public string tipo_de_lente { get; set; } = string.Empty;

        public string material { get; set; } = string.Empty;

        public float precio { get; set; }

        public int cantidad_disponible { get; set; }
    }
}

File: consultorio.client/src/app/protected-app/models/armazon.ts
export interface Iarmazon {
  marca: string;
  modelo: string;
  color: string;
  tipo_de_lente: string;
  material: string;
  precio: number;
  cantidad_disponible: number;
}

export class armazon implements Iarmazon {
  armazonid!: number;
  marca!: string;
  modelo!: string;
  color!: string;
  tipo_de_lente!: string;
  material!: string;
  precio!: number;
  cantidad_disponible!: number;

  constructor() { }

}
